dojo/settings/settings.dist.py

```diff
@@ -1257,6 +1257,7 @@ def saml2_attrib_map_format(dict):
     'Popeye Scan': ['title', 'description'],
     'Wazuh Scan': ['title'],
     'Nuclei Scan': ['title', 'cwe', 'severity'],
+    'Threagile risks report': ['title', 'cwe', 'unique_id_from_tool']
```

Comment:
Since `DEDUPE_ALGO_UNIQUE_ID_FROM_TOOL_OR_HASH_CODE` is being used, the `unique_id_from_tool` field should not be used in the hash code calculation

Reply:
Didn't know that. Will remove.

Reply:
Updated hash code fields list. PTaL